### DISCUSSION TITLE
Invalid Dates

### DISCUSSION THREAD

**Discussion by Jan2Philip** [Digitaler-Impfnachweis/certification-apis#150] [Q&A] [answered]
Hallo,
seit heute kriegen wir bei dem Versuch Genesenenzertifikate zu erzeugen den Fehler "Sorry, there is something invalid in your request: Invalid Dates". Gestern lief noch alles ohne Fehler. Der Fehler kommt unabhängig von den Werten die wir übermitteln (Gültig ab: pos. Test +28 Tage; Gültig bis: pos. Test + 180 Tage/6Monate).

Impfzertifikate lassen sich ohne Probleme ausstellen.

System: Testumgebung (SMCB/KoPs)
Zertifikat: Genesenenzertifikat

**Comment by thinkberg** [comment, marked as answer]
Das ist kein Fehler auf ihrer Seite und wird wieder behoben.

**Reply by Jan2Philip** [reply]
Danke.

**Reply by thinkberg** [reply]
Fixed.

**Reply by Jan2Philip** [reply]
Getestet. Läuft wieder.
